components/utilities/headerLink.js

```diff
@@ -31,9 +31,10 @@ const HeaderLink = ({ name, level, className, children }) => {
   };
 
   // Check if there's a <code> tag in the heading, so we can style it properly.
-  // If we're getting something back, that means there is one, so we add the special class
+  // If we're getting something back, that means there is one, so we will wrap
+  // the header contents in <span> to prevent it from flexing
   const withInlineCode = Array.from(children).find(
-    (child) => child && child.props && child.props.mdxType === "inlineCode"
+    (child) => child && child.type && child.type === "code",
```

Comment:
Nit: there's a new-ish JS syntax that can clean up that line from this:

```
    (child) => child && child.type && child.type === "code",
```

...to this:

```
    (child) => child?.type === "code",
```

Reply:
Updated.